systems/frontier_user_guide.rst

```diff
@@ -2615,21 +2615,60 @@ For a full list of software availability and latest news at the OLCF, please ref
 Debugging
 ============
 
-ARM DDT
+Linaro DDT
 -------
 
```

Comment:
Underline too short on line 2619

Reply:
fixed in 15e4ea35ac8e6c884ee1e03be22a3c76c1d214d3